### PR TITLE
sessionctx/variable: fix default tx_read_only variable value.

### PR DESCRIPTION
**Pull request by coocood**
MySQL show this variable as "ON"/"OFF" in documentation, but the real value returned is  "1"/“0”.

### CODE REVIEW

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
@shenli PTAL

File: sessionctx/variable/sysvar.go [hunk 1]
```diff
@@ -479,7 +479,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeNone, "explicit_defaults_for_timestamp", "OFF"},
 	{ScopeNone, "performance_schema_events_waits_history_size", "10"},
 	{ScopeGlobal, "log_syslog_tag", ""},
-	{ScopeGlobal | ScopeSession, "tx_read_only", "OFF"},
+	{ScopeGlobal | ScopeSession, "tx_read_only", "0"},
```
Comment (ngaut) [inline sessionctx/variable/sysvar.go]: What if client driver check if the value is "OFF" ?

Comment (coocood) [inline sessionctx/variable/sysvar.go]: MySQL never returns values as "OFF", so if a client driver checks if the value is "OFF", it will always fail.

**Comment by ngaut** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**